server/graphql/invoice/src/mutations/customer_return/insert.rs

```diff
@@ -165,3 +175,13 @@ impl CustomerReturnLineInput {
         }
     }
 }
+
+impl InsertCustomerReturnStatusInput {
+    pub fn to_domain(&self) -> InsertCustomerReturnStatus {
+        use InsertCustomerReturnStatus::*;
+        match self {
+            InsertCustomerReturnStatusInput::New => New,
+            InsertCustomerReturnStatusInput::Shipped => Shipped,
```

Comment:
This should be `Verified`, not `Shipped`

Reply:
Nice good catch thank you - I had copied the supplier statuses